sklearn/metrics/classification.py

```diff
@@ -1558,6 +1558,10 @@ def log_loss(y_true, y_pred, eps=1e-15, normalize=True, sample_weight=None):
         Predicted probabilities, as returned by a classifier's
         predict_proba method.
 
+    labels : array-like, optional (default=None)
+        If not None , LabelBinarizer will fit labels instead of y_true
```

Comment:
I would still make this a bit more explicit, "if not provided, labels will be inferred from y_true"